src/public/admin/logs/styles.css

```
@@ -154,31 +156,38 @@
   font-size: 12.5px;
 }
+
 .logs-table tbody td:first-child {
   color: var(--ink-muted);
   font-size: 12px;
   font-family: 'Courier New', monospace;
 }
 
+/* Force long log messages to wrap rather than overflow the table */
+.logs-td--msg {
+  word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)